README.md

```diff
@@ -1,5 +1,19 @@
 # ffd-info-exchange
 
+## About this project
+
+Welcome to the ffd-info-exchange repository! The Federal Front Door: Information Exchange project is a twelve-week discovery engagement meant to uncover possibilities for information sharing between federal agencies as a way of providing better public access to government services. 
```

Comment:
The closest we have to a roadmap _for the alpha_ would be: https://docs.google.com/presentation/d/1Pcdh97yDAJPB3baeqJar5FNoGNSpJRSbHpW8x4CZPpc/edit#slide=id.p